docs/service-brokers/helm-broker-service-classes/gcp-broker/docs/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+type: Overview
+---
+>**NOTE:** You must create a Secret with provider credentials in every Namespace where you provision the GCP Service Broker class. You can provision only one instance of the GCP Service Broker in each Namespace.
+
+The GCP Service Broker class exposes services from the [Google Cloud Platform Service Broker](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker)
+in a given Namespace of the Kyma cluster.
+
+## How to create a secret
```

Comment:
```suggestion
## Create a Secret
```

Reply:
The same question: isn't it documented somewhere so that we'd only link to the external sources?

Reply:
this document is removed